zebra-state/src/error.rs

```diff
@@ -74,4 +74,11 @@ pub enum ValidateContextError {
         difficulty_threshold: CompactDifficulty,
         expected_difficulty: CompactDifficulty,
     },
+
+    #[error("sprout double-spend detected, in finalized state: {in_finalized_state:?}, duplicate nullifier: {nullifier:?}")]
```

Comment:
I think this wording might be better:
```suggestion
    #[error("sprout double-spend: duplicate nullifier: {nullifier:?}, in finalized state: {in_finalized_state:?}")]
```